infra/feast-operator/README.md

```diff
@@ -0,0 +1,11 @@
+# Feast Feature Server Helm-based Operator
+
+Leverages [operator-sdk](https://github.com/operator-framework/operator-sdk) and the [feast-feature-server helm chart](/infra/charts/feast-feature-server).
+
+### To run against an OCP cluster -
```

Comment:
small request, can we swap OCP to K8S here since we're upstream?

Reply:
done

Reply:
thanks!